pages/dkp/kommander/1.3/release-notes/index.md

```diff
@@ -28,7 +27,32 @@ Kommander provides a command center for all your cloud native management needs i
 | **Minimum**        | 1.16.0  |
 | **Maximum**        | 1.18.x  |
 | **Default**        | 1.18.8  |
-# New Features and Capabilities
+
+# Breaking changes
+
+## Docker hub rate limiting 
+Docker Hub announced an update to their image pull policies in August, 2020. The change results in the need to change cluster configurations to accommodate new account structures that enable image pull rate limiting.
+
+Rate limiting happens on a per-pull basis regardless of whether the pulled image is owned by a paid user. This means D2iQ, as owner of most images used in Konvoy, does not have any influence as to whether your current address is rate-limited or not. Konvoy does not have a strict dependency on Docker Hub accounts or plans.
+
+For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
+
+# New Features
+
+## Kudo Spark compatibility 
```

Comment:
Wording from Konvoy: https://github.com/mesosphere/konvoy/pull/2734/files

```
# KUDO Spark compatibility

Kommander 1.3 requires KUDO Spark 3.0 because Spark 2.4 does not support Kubernetes 1.18.
To continue using KUDO Spark on Konvoy 1.6 and above, upgrade to KUDO Spark 3.0.
For details on how to migrate workloads to Spark 3.0, consult the official [migration guides](https://spark.apache.org/releases/spark-release-3-0-0.html) for each relevant module.
```